bluesky/plans.py

```diff
@@ -726,17 +726,31 @@ def inner_scan_nd():
 
 
 def inner_product_scan(detectors, num, *args, per_step=None, md=None):
+    # For scan, num is the _last_ positional arg instead of the first one.
+    # Notice the swapped order here.
+    yield from scan(detectors, *args, num, per_step=None, md=None)
```

Comment:
should we pass through `plan_name='inner_product_scan'`

Reply:
What do you think? I considered it but decided against it.

Reply:
I am +0.5, we should consider these headers part of the API, but `scan` is nicer.

Reply:
OK, done. 👍